src/status_im/contexts/settings/common/header.cljs

```diff
@@ -0,0 +1,16 @@
+(ns status-im.contexts.settings.common.header
```

Comment:
Added this because it's the same header for terms of use, privacy policy and abour screens